<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>864</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>CEGUI Unified Editor</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/application_icon.png</normaloff>icons/application_icon.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QTabWidget" name="tabs">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <property name="usesScrollButtons">
       <bool>true</bool>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>813</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNewFile"/>
    <addaction name="actionOpenFile"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAll"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionLicense"/>
   </widget>
   <widget class="QMenu" name="menuPanels">
    <property name="title">
     <string>Panels</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuProject">
    <property name="title">
     <string>Project</string>
    </property>
    <addaction name="actionNewProject"/>
    <addaction name="separator"/>
    <addaction name="actionOpenProject"/>
    <addaction name="actionSaveProject"/>
    <addaction name="actionCloseProject"/>
    <addaction name="separator"/>
    <addaction name="actionProjectSettings"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuProject"/>
   <addaction name="menuPanels"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="globalToolbar">
   <property name="windowTitle">
    <string>Global Toolbar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewFile"/>
   <addaction name="actionOpenFile"/>
   <addaction name="actionSave"/>
   <addaction name="actionSaveAll"/>
   <addaction name="separator"/>
   <addaction name="actionSaveProject"/>
   <addaction name="actionProjectSettings"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
  </widget>
  <action name="actionQuit">
   <property name="icon">
    <iconset>
     <normaloff>icons/actions/quit.png</normaloff>icons/actions/quit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionCloseProject">
   <property name="icon">
    <iconset>
     <normaloff>icons/actions/close_project.png</normaloff>icons/actions/close_project.png</iconset>
   </property>
   <property name="text">
    <string>Close project</string>
   </property>
  </action>
  <action name="actionOpenProject">
   <property name="icon">
    <iconset>
     <normaloff>icons/actions/open_project.png</normaloff>icons/actions/open_project.png</iconset>
   </property>
   <property name="text">
    <string>Open project</string>
   </property>
  </action>
  <action name="actionNewProject">
   <property name="icon">
    <iconset>
     <normaloff>icons/actions/new_project.png</normaloff>icons/actions/new_project.png</iconset>
   </property>
   <property name="text">
    <string>New project</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normaloff>icons/actions/save.png</normaloff>icons/actions/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAll">
   <property name="icon">
    <iconset>
     <normaloff>icons/actions/save_all.png</normaloff>icons/actions/save_all.png</iconset>
   </property>
   <property name="text">
    <string>Save all</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset>
     <normaloff>icons/actions/close.png</normaloff>icons/actions/close.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset>
     <normaloff>icons/actions/undo.png</normaloff>icons/actions/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset>
     <normaloff>icons/actions/redo.png</normaloff>icons/actions/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionSaveProject">
   <property name="icon">
    <iconset>
     <normaloff>icons/actions/save_project.png</normaloff>icons/actions/save_project.png</iconset>
   </property>
   <property name="text">
    <string>Save project</string>
   </property>
  </action>
  <action name="actionProjectSettings">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/actions/project_settings.png</normaloff>icons/actions/project_settings.png</iconset>
   </property>
   <property name="text">
    <string>Project Settings</string>
   </property>
  </action>
  <action name="actionLicense">
   <property name="text">
    <string>License</string>
   </property>
  </action>
  <action name="actionNewFile">
   <property name="icon">
    <iconset>
     <normaloff>icons/actions/new_file.png</normaloff>icons/actions/new_file.png</iconset>
   </property>
   <property name="text">
    <string>New file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="icon">
    <iconset>
     <normaloff>icons/actions/open_file.png</normaloff>icons/actions/open_file.png</iconset>
   </property>
   <property name="text">
    <string>Open file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionRecentProjects">
   <property name="text">
    <string>Recent Projects</string>
   </property>
  </action>
  <action name="actionRecent_projects">
   <property name="text">
    <string>Recent projects</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
